src/app.js

```diff
@@ -197,11 +199,14 @@ export function app(app) {
     function removeChild() {
       parent.removeChild(element)
     }
+    recoverNode(node)
```

Comment:
after looking at the chrome devtools memory I think that these calls for recoverNode(oldChild) and recoverNode(reusableNode) are not proper and infact cause the extra js heap bloat in that memory graph I shared... heh